packages/pwa-kit-react-sdk/package.json

```diff
@@ -156,6 +157,7 @@
     "@loadable/component": "^5.15.0",
     "@loadable/server": "^5.15.0",
     "@loadable/webpack-plugin": "^5.15.0",
+    "cosmiconfig": "^7.0.1",
```

Comment:
curious why is this a peerdependency?

Reply:
I kinda agree with Kevin – if the PWA doesn't import `cosmiconfig` directly (doesn't look like it to me), then I'd expect `cosmicconfig` to be a `dependency`. Calling code in the `pwa` would then only touch our `getConfig` function...

Reply:
Totally agree with both of you. Surprisingly doing this solved a problem I was having. I was waiting on review to get confirmation that this was not what I wanted. So I'll go back and give it another attempt as getting it working as a normal dependency.